.github/2.0/cookbooks/Flow.md

```diff
@@ -431,6 +431,92 @@ f = (Flow()
 
 </table>
 
+#### Commonly used arguments for distribution setup in `.add`
+
+| Name | default | Description |
+| --- | --- | --- |
+| `host` | `0.0.0.0` | The host of the machine. Can be an ip address or DNS name (e.g. `0.0.0.0`, `my_encoder.jina.ai`) |
+| `port_expose` | randomly initialized | Port of JinaD on the remote machine. |
```

Comment:
Are we sure about this? I think this is only for `gateway` and is the port_exposed by it.

Reply:
@deepankarm

Reply:
But it does not default to 8000. It is randomly initialized. :D I think we need to clarify this in code and ideally split `port_expose` into two concepts:

1. The port the client can speak to
2. The port that JinaD exposes

Reply:
I suggest we do the renaming of the parameters in another PR.